source/issues-and-help.txt

```diff
@@ -0,0 +1,50 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant MongoDB C#/.NET community
+that includes users with varying levels of experience using
+the driver. We find the quickest way to get support for general
+questions is through the `MongoDB Community Forums
+<https://www.mongodb.com/community/forums>`__.
+
+To learn more, refer to our :manual:`support channels </support>`.
```

Comment:
Suggestion:

A couple items here:
1) When cross-linking, make sure it's specific about what page it's linking to (see [Links and Cross Reference](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-text)s in the style guide) 
"To learn more about MongoDB support options..."

2) The Technical Support page links to the forums and general, non-driver specific MongoDB resources. The intro text for the forums is duplicated which is a good indicator there's redundant text. One solution could be to omit this link (what I would suggest) and another could be to focus on describing the Technical Support page instead of repeating the content of one of its sections.

Reply:
chose to omit the link. I see that the technical support page links right back to the community forums.